src/main/webapp/app/registry/encryption/encryption.service.ts

```diff
@@ -8,10 +8,10 @@ export class EncryptionService {
   constructor(private http: HttpClient) {}
 
   encrypt(textToEncrypt: string): Observable<string> {
-    return this.http.post<string>('config/encrypt', textToEncrypt).pipe(map((response: string) => '{cipher}' + response));
+    return this.http.post('config/encrypt', textToEncrypt, { responseType: 'text' }).pipe(map((response: string) => '{cipher}' + response));
```

Comment:
if you want to specify the type as `<string>`, then, you can use the syntax as `responseType: 'text' as 'json'`

Reply:
Is there a preference?  That's another one of the solutions in the linked StackOverflow but seems a little hacky (so does removing the type though).

Reply:
both works just fine. lets go ahead with current solution